README.md

```diff
@@ -31,6 +31,7 @@ Flags:
   -f, --filters strings                      Comma-separated list of filters in format "type=name". ex: elasticsearch=my-cluster (Supported types [agent apm beat elasticsearch enterprisesearch kibana maps])
   -h, --help                                 help for eck-diagnostics
       --kubeconfig string                    optional path to kube config, defaults to $HOME/.kube/config
+  -l, --log-selectors stringArray            Label selectors to restrict the logs to be collected. Can be specified more than once.
```

Comment:
nit: not sure if `stringArray` helps to understand what is expected, let's maybe add an example here?
```suggestion
  -l, --log-selectors string            Label selectors to restrict the logs to be collected. You can also use multiple flag options like -l 'elasticsearch.k8s.elastic.co/node-master=true,elasticsearch.k8s.elastic.co/node-data!=true' -l common.k8s.elastic.co/type=kibana
```

Reply:
Yeah good point. That `stringArray` is autogenerated. I think giving examples makes sense.